scripts/install_spack.sh

```diff
@@ -96,6 +96,7 @@ sed \
 
 cp ${PAWSEY_SPACK_CONFIG_REPO}/scripts/templates/spack_refresh_modules.sh \
    ${PAWSEY_SPACK_CONFIG_REPO}/scripts/templates/spack_rm_modules.sh \
+   ${PAWSEY_SPACK_CONFIG_REPO}/scripts/templates/spack_generate_migration_scripts.sh \
```

Comment:
The script is not currently located in the `template` subdirectory.

As a side note we should consider creating a better separation between scripts to be installed with spack and scripts run to deploy spack (and the stack). Maybe we should rename `templates` to something else. This can be addressed in another PR, I think the nest solution now is to move the script in `templates` for coherency.